lib/Comment/CommentMeta.js

```diff
@@ -1,41 +1,63 @@
-import React from 'react';
-import { func, string } from 'prop-types';
+import React, { Fragment } from 'react';
+import { bool, func, string } from 'prop-types';
 import Button from '../Button';
 
-export function CommentMeta({ createdAt, onEditClick, onRemoveClick }) {
+export function MetaDot() {
+  return <span className="text-xs text-neutral-primary mr-1 relative">•</span>;
+}
```

Comment:
A tiny helper component for displaying that dot that appears in places on comments!